cfn-resources/org-invitation/README.md

```diff
@@ -0,0 +1,13 @@
+# Mongodb::Atlas::OrgInvitation
+
```

Comment:
Copy format of Cluster README file. Also missing links to Examples section + Link to Open API Docs for reference at bottom.

Reply:
Updated the documentation